src/components/rockets/Rockets.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Rockets from './Rockets';
+import store from '../../redux/configureStore';
+
+const MockRockets = () => (
+  <Provider store={store}>
+    <Rockets />
+  </Provider>
+);
+
+test('renders correctly', () => {
+  const tree = renderer.create(<MockRockets />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Great work using react-test-renderer to test components. Kindly write tests for this component using React Testing Library just the same way you did for the `Mission` component. 👍